.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }} [ ${{ matrix.os }} ]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python: [3.8]
+        toxenv: [test]
+        toxargs: [-v]
+
+        include:
+
+          # - name: Tests that all the basics are covered
```

Comment:
I'd remove this commented-out example - I don't think it adds anything to the real ones below.

Reply:
OK, will do it.

Reply:
done